Automattic/wp-calypso
# Plugins marketplace: Change formatting of search and filtering

Related to https://github.com/Automattic/dotcom-forge/issues/7537

## Proposed Changes

* Update the design of the search and filters on `/plugins` to the one proposed on the linked issue.
* Figma: https://www.figma.com/design/kyCC1EHKtv4qsElmlAzp5T/Flyout-panel-i2?node-id=537-123318&m=dev

## Testing Instructions

* Open the live preview and go to `/plugins`
* Check that the search and filters still work, and look like the proposed design

## Pre-merge Checklist
